samples/nrf_desktop/src/modules/ble_bond.c

```diff
@@ -328,7 +349,66 @@ static void silence_unused(void)
 	}
 }
 
-static void init(void)
+static int settings_set(int argc, char **argv, void *val_ctx)
+{
+	if (argc != 1) {
+		return -ENOENT;
+	}
+
+	if (!strcmp(argv[0], PEER_ID_STORAGE_NAME)) {
+		int len = settings_val_read_cb(val_ctx, &cur_peer_id,
+					       sizeof(cur_peer_id));
+		if (len != sizeof(cur_peer_id)) {
+			LOG_ERR("Can't read peer id from storage");
+			return len;
+		}
+	}
+
+	return 0;
+}
+
+static int commit(void)
+{
+	/* Update peer id - select, then selected (for proper LED update) */
+	struct ble_peer_operation_event *event =
+		new_ble_peer_operation_event();
+
+	event->op = PEER_OPERATION_SELECT;
```

Comment:
it's better to fix leds than sending two events

Reply:
ok, so I will make proper changes led_state